tactics/hints.ml

```diff
@@ -1628,7 +1628,7 @@ let pr_id_hint env sigma (id, v) =
   | Some (ConstrPattern p | SyntacticPattern p) -> str", pattern " ++ pr_lconstr_pattern_env env sigma p
   | Some DefaultPattern -> str", pattern " ++ pr_leconstr_env env sigma (get_default_pattern v.code.obj)
   in
-  (pr_hint env sigma v.code ++ str" (level " ++ int v.pri ++ pr_pat v
+  (pr_hint env sigma v.code ++ str" (pri " ++ int v.pri ++ pr_pat v
```

Comment:
My examination of the code shows that "pri(ority)" is only used to set the order of the hintDb entries, which seems a more informative word than "level".  The documentation uses to "cost" in several places, but AFAICT the code in `auto` and `typeclasses eauto` doesn't even look at the pri field in the code, so they can't be adding these values and comparing to a limit.  The real limit is the number of proof steps.

Reply:
Maybe "priority" would be clearer in the output than "pri"?